tests/ci/azure_pipeline_test/dsvm_trigger_linux_multistage.yml

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
where is this used?

Reply:
https://dev.azure.com/AZGlobal/Azure%20Global%20CAT%20Engineering/_build?definitionId=134&_a=summary

we can create a pipeline off of it in the other ado as well, but I didnt want to do to much there without everyone on the team understanding it first

Reply:
![image](https://user-images.githubusercontent.com/9027725/70840094-1fb32200-1dde-11ea-806b-34214fb6e3ca.png)